sdk/cosmosdb/cosmos/src/client/SasToken/PermissionScopeValues.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Represents permission scope values.
+ */
+export enum PermissionScopeValues {
```

Comment:
Here is a more idiomatic way of doing this without an enum: https://github.com/Azure/azure-sdk-for-js/commit/a5f81d843dadccb257d8bdd24a0b6d8685a40f85#diff-14c03dea068f45632f6d792a43a7683bfe6afe584fae760d6334c3e574072663